### PR TITLE
Task: automate update loc files

### PR DESCRIPTION
**Pull request by thewahome**
Creates a PR into the [GE](https://github.com/microsoftgraph/microsoft-graph-explorer-v4) repository with the changes that have been updated to json files within the messages folder

image

### CODE REVIEW

File: .github/workflows/handover-translations.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Hand over tranlation files
```
Comment (irvinesunday) [inline .github/workflows/handover-translations.yml:1]:
```suggestion
name: Hand over translation files
```

File: .github/workflows/handover-translations.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - name: Create pull request to handover translations
+      uses: paygoc6/action-pull-request-another-repo@v1.0.1
+      env:
+        API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
+      with:
+        source_folder: 'messages/translated-content/**'
+        destination_repo: 'microsoftgraph/microsoft-graph-explorer-v4'
+        destination_folder: 'src/messages'
+        destination_base_branch: 'dev'
+        destination_head_branch: handover-translations/${{ env.today }}
```
Comment (irvinesunday) [inline .github/workflows/handover-translations.yml:35]: This is a curious question: Why do you need to create differently named branches, named by dates? From my experience with the permissions-scraper (content-assistant) bot, new incoming commits will just get added to an existing PR if it is still unmerged.

Comment (thewahome) [reply .github/workflows/handover-translations.yml:35]: This makes sense.. There were no strong factors to consider when naming of the branch.

File: .github/workflows/handover-translations.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    paths:
+      - 'messages/**'
+      - '!messages/GE.json'
+
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
```
Comment (MaggieKimani1) [inline .github/workflows/handover-translations.yml:16]: - uses: actions/checkout@v2.3.4

**Review by MaggieKimani1 — CHANGES_REQUESTED** [review]
- uses: actions/checkout@v2.3.4

**Review by MaggieKimani1 — APPROVED** [review]

**Pull request merged**